pkgs/os-specific/linux/firmware/rtlwifi_new-firmware/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, linuxPackages }:
+
+with lib;
+
+stdenv.mkDerivation {
+  name = "rtlwifi_new-firmware-${linuxPackages.rtlwifi_new.version}";
+  inherit (linuxPackages.rtlwifi_new) src;
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
I just want to copy firmware files from repository to nix store without building phase. What the best way to achieve it?